src/dex-ui/pages/dao/CreateADAO/forms/CreateTokenDAOGovernanceForm.tsx

```diff
@@ -0,0 +1,326 @@
+                ...register("governance.newToken.initialSupply", {
+                  required: { value: true, message: "An initial token supply is required." },
+                }),
+              },
+            }}
+            formHelperText="If not specified, can be unlimited"
+            isInvalid={Boolean(errors.governance?.newToken?.initialSupply)}
+            errorMessage={
+              errors.governance?.newToken?.initialSupply && errors.governance?.newToken?.initialSupply.message
+            }
+          />
+          <Flex direction="row" gap="4" alignItems="flex-end">
+            <FormInput<"governance.newToken.supplyKey">
+              flex="5"
+              inputProps={{
+                flex: 5,
+                id: "governance.newToken.supplyKey",
+                label: "Supply Key",
+                type: "text",
+                isReadOnly: true,
+                placeholder: "Enter Supply Key",
+                register: {
+                  ...register("governance.newToken.supplyKey", {
+                    required: { value: true, message: "Supply key is required." },
+                  }),
+                },
+              }}
+              isInvalid={Boolean(errors.governance?.newToken?.supplyKey)}
+              errorMessage={errors.governance?.newToken?.supplyKey && errors.governance?.newToken?.supplyKey?.message}
+            />
+            <Button key="token-supply-key" onClick={handleGetAccountInfoClick} flex="1" marginBottom="8px">
+              Get Supply Key
+            </Button>
+          </Flex>
+        </DAOFormContainer>
+        <Text textStyle="p large regular">Initial token distribution</Text>
+        <DAOFormContainer>
+          <FormInput<"governance.newToken.treasuryWalletAccountId">
+            inputProps={{
+              id: "governance.newToken.treasuryWalletAccountId",
+              label: "Treasury wallet account id",
+              type: "text",
+              placeholder: "Enter wallet account id",
+              isReadOnly: isFormInReadOnlyMode,
+              register: {
+                ...register("governance.newToken.treasuryWalletAccountId", {
+                  required: { value: true, message: "A treasury account id is required." },
+                }),
+              },
+            }}
+            isInvalid={Boolean(errors.governance?.newToken?.treasuryWalletAccountId)}
+            errorMessage={
+              errors.governance?.newToken?.treasuryWalletAccountId &&
+              errors.governance?.newToken?.treasuryWalletAccountId.message
+            }
+          />
+        </DAOFormContainer>
+        <Flex gap="4" alignItems="center">
+          <Notification
+            type={NotficationTypes.WARNING}
+            textStyle="b3"
+            message={`If you create a new token, 
+            token Id input will get up popup automatically after successful new token creation.`}
+          />
+          <Button key="create-token" onClick={createNewToken}>
```

Comment:
I'm getting the following error after creating a new token:
![Screen Shot 2023-05-17 at 9 21 42 PM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/1d100f99-e400-412b-ad56-77432005549b)

Reply:
I am not sure why are you getting this error I tested on my machine also with Anshul, may be after standup today we can quickly again check this together to get the root cause of the issue. I was able to create multiple tokens but never got this issue.